.drone.star

```diff
@@ -1609,6 +1618,60 @@ def notify(ctx):
         },
     }
 
+def ocisServerWithIdp():
+    environment = {
+        "GRAPH_IDENTITY_BACKEND": "ldap",
+        "GRAPH_LDAP_SERVER_WRITE_ENABLED": "true",
+        "LDAP_URI": "ldaps://0.0.0.0:9235",
+        "LDAP_INSECURE": "true",
+        "LDAP_BIND_DN": "uid=libregraph,ou=sysusers,o=libregraph-idm",
+        "LDAP_BIND_PASSWORD": "idm",
+        "LDAP_USER_BASE_DN": "ou=users,o=libregraph-idm",
+        "LDAP_USER_SCHEMA_ID": "ownclouduuid",
+        "LDAP_USER_SCHEMA_MAIL": "mail",
+        "LDAP_USER_SCHEMA_USERNAME": "uid",
+        "LDAP_USER_OBJECTCLASS": "inetOrgPerson",
+        "LDAP_GROUP_BASE_DN": "ou=groups,o=libregraph-idm",
+        "LDAP_GROUP_SCHEMA_ID": "ownclouduuid",
+        "LDAP_GROUP_SCHEMA_MAIL": "mail",
+        "LDAP_GROUP_SCHEMA_GROUPNAME": "cn",
```

Comment:
```suggestion
        "LDAP_GROUP_SCHEMA_GROUPNAME": "cn",
        "LDAP_GROUP_SCHEMA_MEMBER": "member",
```

This should fix a couple of group sharing related failures e.g. `tests/acceptance/features/apiWebdavUploadTUS/uploadToShare.feature:66`